### ISSUE TITLE
Wrong markdown when convert bold text

### ISSUE THREAD

**Issue by tiennampham23** [crosstype/node-html-markdown#14]
Hi Ron, first of all, I am very excited about your library, it's so fast. But maybe it has a mirror bug.

## My example HTML:
![image](https://user-images.githubusercontent.com/26276909/123904122-27272580-d99a-11eb-8b18-41a02f153a74.png)

```
<p><strong style="color: rgb(51, 51, 51);">1944:&nbsp;</strong><span style="color: rgb(51, 51, 51);">Maren Larsen is a blonde beauty from a small Minnesota farming town.</span></p>
```

## My Markdown output:
```
**1944:&nbsp;**Maren Larsen is a blonde beauty from a small Minnesota farming town.
```

You can see this output, Beside "1994" that has a non-break space. So it's a bold style of this year will be not worked properly.

## Expect result:
```
**1944**Maren Larsen is a blonde beauty from a small Minnesota farming town.
```

I tried to use the other libraries like turndown and it's worked properly. I love your library and so, I create an issue to report to you.  I hope you can consider this issue and reply to me. Thank you

**Comment by nonara** [comment]
Thanks for the report and the kind words, Tien! I'm glad you like the library.

Looks like there is an upstream issue in the parser. I didn't realize that text nodes weren't being properly decoded. I have a PR submitted for the parser and will make sure this is cleared up when it's done.

To me, this would be the appropriate output:

```
**1944:** Maren Larsen is a blonde beauty from a small Minnesota farming town.
```

Which would display as:

**1944:** Maren Larsen is a blonde beauty from a small Minnesota farming town.

When the PR closes, I'll ensure that everything is good and bump the parser version.

**Comment by nonara** [comment]
Update: Upstream fix is ready. I will be addressing this ASAP.

**Comment by tiennampham23** [comment]
Awesome, thank you!

**Comment by nonara** [comment]
Fixed in [v1.0.0](https://github.com/crosstype/node-html-markdown/releases/tag/v1.0.0).
